isacc_messaging/api/views.py

```diff
@@ -202,3 +202,39 @@ def update_patient_extensions(dry_run):
         patient = Patient(json_patient)
         patient.mark_next_outgoing(persist_on_change=not dry_run)
         patient.mark_followup_extension(persist_on_change=not dry_run)
+
+
+@base_blueprint.cli.command("maintenance-reinstate-all-patients")
+def update_patient_params():
+    """Iterate through all patients, activate all of them"""
+    from isacc_messaging.models.fhir import next_in_bundle
+    from isacc_messaging.models.isacc_patient import IsaccPatient as Patient
+    all_patients = Patient.all_patients()
+    for json_patient in next_in_bundle(all_patients):
+        patient = Patient(json_patient)
+        patient.active = True
+        patient.persist()
+        audit_entry(
+        f"Patient with id {patient.id} reinstated",
```

Comment:
audit's are intended for paper trail.  i'd expect this CLI is typically only going to be used once - when this feature is turned on.  i'd suggest "Patient {patient.id} active set true" or the like.